sdk/core/core-rest-pipeline/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Features Added
 
+- Move `defaultLogPolicy` to after `Sign` phase in `createPipelineFromOptions`. [#22930](https://github.com/Azure/azure-sdk-for-js/pull/22930)
```

Comment:
nit: I think this should probably be in the `Other Fixes` section

Reply:
Done!